data.sql

```diff
@@ -4,3 +4,15 @@ INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALU
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Gabumon', '2018/11/15', 2, 'true', 8.0);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Pikachu', '2021/7/1', 1, 'false', 15.04);
 INSERT INTO animals (name,date_of_birth,escape_attempts,neutered,weight_kg) VALUES ('Devimon', '2017/12/5', 5, 'true', 11.0);
+
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES
+    ('Charmander', '2020-02-08', 0, false, 11.0),
```

Comment:
👏 Great work! You've been making excellent progress on your project. However, there are a couple of important updates needed:

- [ ] As per the project requirement, the `weight_kg` value for "Charmander" should be `-11.0`. In your current code, you have it as `11.0`, which doesn't align with the project requirement. To ensure your data is accurate and in accordance with the project requirements, kindly update your insert query to set the "weight_kg" for "Charmander" to `-11.0`.

- [ ] Additionally, after updating your insert query, please make sure to run all the queries in the `queries.sql` file. This will help ensure that your database contains the correct data as per the project requirements. Don't forget to update the screenshots in the PR Description if necessary to reflect the changes.

Keep up the great work, and thank you for your dedication to this project! 👍🔧🐾

Reply:
** Changed the weights of animals**
-
![image](https://github.com/AbuTalha3/vet-clinic-database/assets/127955219/39dc25fc-32a7-4b6e-94e9-dc2c7bf3b29a)
-
![image](https://github.com/AbuTalha3/vet-clinic-database/assets/127955219/a2752d29-45f0-448c-bedc-d2f3fd81c737)